ketcher-autotests/tests/Macromolecule-editor/RNA-Builder/rna-library.spec.ts

```diff
@@ -1378,4 +1383,165 @@ test.describe('RNA Library', () => {
 
     await page.getByTestId('cancel-btn').click();
   });
+
+  enum MonomerLocationTabs {
+    PEPTIDES = 'Peptides',
+    PRESETS = 'Presets',
+    SUGARS = 'Sugars',
+    BASES = 'Bases',
+    PHOSPHATES = 'Phosphates',
+    NUCLEOTIDES = 'Nucleotides',
+    CHEM = 'CHEM',
+  }
+
+  interface IIDTSearchString {
+    testDescription: string;
+    IDTSeatchString: string;
+    // Location where searched monomer located (we have to go to that location to make sure it is where)
+    ResultMonomerLocationTab: MonomerLocationTabs;
+    // Set shouldFail to true if you expect test to fail because of existed bug and put issues link to issueNumber
+    shouldFail?: boolean;
+    // issueNumber is mandatory if shouldFail === true
+    issueNumber?: string;
+    // set pageReloadNeeded to true if you need to restart ketcher before test (f.ex. to restart font renderer)
+    pageReloadNeeded?: boolean;
+  }
+
+  async function goToMonomerLocationTab(
+    page: Page,
+    monomerLocation: MonomerLocationTabs,
+  ) {
+    switch (monomerLocation) {
+      case 'Peptides':
+        await goToPeptidesTab(page);
+        break;
+      case 'Presets':
+        await goToRNATab(page);
+        // Presets tab openned by default
+        break;
+      case 'Sugars':
+        await goToRNATab(page);
+        await toggleSugarsAccordion(page);
+        break;
+      case 'Bases':
+        await goToRNATab(page);
+        await toggleBasesAccordion(page);
+        break;
+      case 'Phosphates':
+        await goToRNATab(page);
+        await togglePhosphatesAccordion(page);
+        break;
+      case 'Nucleotides':
+        await toggleNucleotidesAccordion(page);
+        break;
+      case 'CHEM':
+        await goToCHEMTab(page);
+        break;
+      default:
+        await goToRNATab(page);
+        break;
+    }
+  }
+
+  async function searchMonomerBuName(page: Page, monomerName: string) {
```

Comment:
The `searchMonomerBuName` function should probably be called `searchMonomerByName` ?